.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules/
+.vscode/
+.idea/
+.DS_Store
```

Comment:
It also may be prudent to add these lines to your [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).